users.json

```diff
@@ -85,7 +85,8 @@
         "tswast",
         "vishald123",
         "AlisskaPie",
-        "tritone"
+        "tritone",
+	"sofisl"
```

Comment:
we should add a linter so that this would be caught automatically (perhaps something like [this](https://github.com/zaach/jsonlint), but mind tabbing `sofisl` in a couple more characters so it matches the spacing of `tritone`; this is the sort of nit I usually try to not make in code review, please forgive me :laughing:

Reply:
no problem! I can make a new PR, it's good practice!